apps/src/templates/sectionAssessments/FreeResponsesSurveyTable.jsx

```diff
@@ -0,0 +1,99 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import i18n from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+
+export const COLUMNS = {
+  RESPONSE: 0,
+};
+
+const freeResponsesDataPropType = PropTypes.shape({
+  id:  PropTypes.number,
+  studentId: PropTypes.string,
+  name: PropTypes.string,
+  response: PropTypes.string,
+});
+
+class FreeResponsesSurveyTable extends Component {
+  static propTypes= {
+    freeResponses:PropTypes.arrayOf(freeResponsesDataPropType),
+  };
+
+  state = {
+    [COLUMNS.RESPONSE]: {
+      direction: 'desc',
+      position: 0
+    }
+  };
+
+  getSortingColumns = () => {
+    return this.state.sortingColumns || {};
+  };
+
+  onSort = (selectedColumn) => {
```

Comment:
No need to address here, but this code is duplicated across our tables. For example the StatsTable has similar sorting functionality.  Maybe worth extracting and re-using with our table styles at some point?